api/src/plugins/fastify-sentry.ts

```diff
@@ -0,0 +1,23 @@
+import { init, captureException } from '@sentry/node';
+import { FastifyInstance, FastifyPluginOptions } from 'fastify';
+import fp from 'fastify-plugin';
+
+const fastifySentry = (
+  fastify: FastifyInstance,
+  options: FastifyPluginOptions,
+  done: (err?: Error) => void
+) => {
+  init(options);
+
+  fastify.setErrorHandler((error, request, reply) => {
+    reply.log.error(error.message);
+    captureException(error);
+    if (error.statusCode)
+      reply.statusCode = error.statusCode >= 400 ? error.statusCode : 500;
```

Comment:
```suggestion
```
To set the status code, you need to call `reply.code` or `reply.status`, but let's keep this PR focused on Sentry.